app/controllers/subscriber_lists_controller.rb

```diff
@@ -39,9 +39,15 @@ def subscriber_list_params
       title: title,
       slug: slug,
       signon_user_uid: current_user.uid,
+      type: type
     )
   end
 
+  def type
+    combine_mode = params[:combine_mode].present? && params[:combine_mode] == "or"
+    (combine_mode ? 'OrJoinedFacetSubscriberList' : '')
```

Comment:
Are the parentheses needed here?  (I don't recall seeing them in the last iteration of this PR)

Reply:
I got paranoid but removed it and the tests still pass, good spot